spec/lib/support_ticket_creator_spec.rb

```diff
@@ -33,6 +33,20 @@
 
       expect { SupportTicketCreator.call(messy_args) }.to_not raise_exception
     end
+
+    it "Raises an error when 422 is returned" do
```

Comment:
Minor: The "R" in Raises should be lowercase.

Reply:
Ah yes. Thanks! fixed.